facetapi.module

```diff
@@ -5,6 +5,16 @@
  * An abstracted facet API that can be used by various search backends.
  */
 
+ /**
+  * A bit flag used to let us know if an object is in the database.
+  */
+ define('EXPORT_IN_DATABASE', 0x01);
```

Comment:
Probably not necessary to do immediately, but we could get rid of the CTools concepts here. Backdrop already has CMI, we don't need to be differentiating between db and code storage, right?